docs/getting-started/using-ale/isd-generate.md

```diff
@@ -0,0 +1,356 @@
+
+    ```sh
+    isd_generate -n -k mro_B10_72W.tm B10_013341_1010_XN_79S172W.cub
+    ```
+
+## Troubleshooting
+
+!!! bug "Troubleshooting Checklist"
+
+    - [ ] Is your image in ISIS format (or PDS format for Pds3Label Drivers)?
+        - [use](../../getting-started/using-isis-first-steps/locating-and-ingesting-image-data.md#introduction-to-importing) 
+          a [`-2isis` app](https://isis.astrogeology.usgs.gov/Application/index.html) to convert images to ISIS format.
+    - [ ] Do you have spice data...
+        - If using a NaifSpice Driver:
+            - [ ] Spice Data [Downloaded from NAIF](#downloading-naif-spice-data-with-wget)?
+            - [ ] [`$ALESPICEROOT` set](#setting-alespiceroot)?
+            - [ ] [Metakernel `PATH_VALUE` set](#setting-the-path-value-in-a-metakernel)?
+        - If using an IsisSpice Driver:
+            - [ ] Spice Data attached to your image by [`spiceinit`](https://isis.astrogeology.usgs.gov/Application/presentation/Tabbed/spiceinit/spiceinit.html)?
+    - [ ] Check what's wrong with a `--verbose` run.
+
+-----
+### Investigating with --verbose
+
+Use the `--verbose` (`-v`) argument to troubleshoot and see why ALE might not be recognizing your image.
+Check for info about the failure under `Trying <class 'ale.drivers.[X]_drivers...'>` where [X] is 
+ALE's abbreviation for the spacecraft/sensor that captured your image.
+
+```sh
+isd_generate -v B10_013341_1010_XN_79S172W.cub
+```
+
+If requesting assistance, it may be helpful to post what version of ALE you have, shown with:
+
+```sh
+isd_generate --version
+```
+
+-----
+
+## Setting Up NAIF Data
+
+### Downloading NAIF SPICE Data with wget
+
+!!! warning ""
+
+    [[NAIF SPICE Data Home](https://naif.jpl.nasa.gov/naif/data.html)] 
+    [[Getting Spice Kernels from the NAIF Server](https://naif.jpl.nasa.gov/pub/naif/toolkit_docs/Tutorials/pdf/individual_docs/34_naif_server.pdf)]
+
+    If you have enough space, NAIF recommends downloading the entire dataset for your spacecraft, detailed below.
+    Otherwise, you can try subsetting the data as detailed in the [NAIF Server tutorial](https://naif.jpl.nasa.gov/pub/naif/toolkit_docs/Tutorials/pdf/individual_docs/34_naif_server.pdf).
+    You will need `wget`, a common download utility.
+
+    Let's download the Clementine archive as a test.
+
+    1.  Find your mission on the [PDS SPICE Archives](https://naif.jpl.nasa.gov/naif/data_archived.html) and copy the **Archive Link**.
+
+        -  The url for the Clementine archive is:    
+           `https://naif.jpl.nasa.gov/pub/naif/pds/data/clem1-l-spice-6-v1.0/clsp_1000`
+
+    1.  `cd` into you spice folder, the folder you have or will set as `$ALESPICEROOT`.
+
+    1.  Piece together the `wget` command.
+        1.  Starting with this base command... (Don't run it yet!)    
+            `wget -m -nH --cut-dirs=5 -R 'index.html*'`
+
+        1.  add the url...  
+            `-nv https://naif.jpl.nasa.gov/pub/naif/pds/data/clem1-l-spice-6-v1.0/`
```

Comment:
We should link to the instructions on ISIS readme on how to download isis data using downloadIsisData.py